Documentation/PageTranslations/Index.rst

```diff
@@ -29,9 +29,9 @@ The form with the fields for the translated page will then be displayed:
 
     Editing the page fields for an alternative page language
 
-Most of the fields are the same as for the page of the default language. Below
+All of the fields are the same as for the page of the default language. Below
```

Comment:
```suggestion
Most of the fields are the same as for the page of the default language. Below
```

Actually the fields "backend_layout" and a few other ones are only available for the default language. So I think we need to keep "most" here.

Reply:
Well, mention that some fields are only visible for the default language.

Reply:
I think that what "most" achieves here, I don't think we need to make it longer than before. Just important that "all" is not right.